0_ImportanceSamplingEnvMaps/main.cpp

```diff
@@ -25,7 +25,7 @@ static core::smart_refctd_ptr<ICPUBuffer> computeLuminancePdf(smart_refctd_ptr<I
 	const core::vector2d<uint32_t> pdfDomainExtent = { envmapExtent.X, envmapExtent.Y };
 
 	const size_t outBufferSize = pdfDomainExtent.X * pdfDomainExtent.Y * sizeof(double);
-	core::smart_refctd_ptr<ICPUBuffer> outBuffer = core::make_smart_refctd_ptr<ICPUBuffer>(outBufferSize);
+	core::smart_refctd_ptr<ICPUBuffer> outBuffer = ICPUBuffer::create({ outBufferSize });
```

Comment:
can you use c++20 designated initializers? where you type out the name of the member you're supposed to be assigning to?